### DISCUSSION TITLE
Unclear note in the documentation

### DISCUSSION THREAD

**Discussion by Deuchnord** [bolt/core#2776] [Q&A]
Hello

While reading the [introduction page about Content Types](https://docs.bolt.cm/4.0/contenttypes/intro) in the documentation, I found this note:

> Note: You should always ensure that the key that defines each of the ContentTypes and the value of the `name` are the same. In the above example, we start with the key `news:`, and the name is set to `name: News`. If these do not match, Bolt will show an error message.

I was wondering what "same" means here exactly, since I have successfully made a ContentType with a `name` attribute different from its key:

```yaml
redirection:
    name: Redirections
    singular_name: Redirection
     # ...
```

Even if the comparison is case-insensitive, both the key and the `name` attribute are different due to the leading _s_ on _Redirections_, and still, the website works perfectly on my side.

**Comment by bobdenotter** [comment]
Ah, yes! It's actually meant to be like "If the key is the same as the (sluggified version of the) `name:` …" 

So, in your case it's suggesting to change `redirection:` to `redirections:` to keep it consistent. If you're fine with it as-is, you can leave it. It _will_ work, where it might go wrong is if you're trying to fetch content.. Will it be `{% setcontent foo = 'redirection' … %}` or `{% setcontent foo = 'redirections' … %}`? Keeping it consistent will make that possible confusion go away. :-)
